sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/configuration-reference.md

```diff
@@ -36,14 +36,15 @@ Configuration Reference:
 
 #### Schema Inference Config
 
-Note when running queries unless if are interested to get back the raw json payload 
-we recommend setting `spark.cosmos.read.inferSchemaEnabled` to be `true`. 
+When doing read operations, users can specify a custom schema or allow the connector to infer it. Schema inference is enabled by default.
 
 | Config Property Name      | Default | Description |
 | :---        |    :----   |         :--- | 
-| `spark.cosmos.read.inferSchemaEnabled`     | `false`    | Whether schema inference is enabled or should return raw json. We recommend to enable this option for query  |
-| `spark.cosmos.read.inferSchemaQuery`      | `SELECT * FROM r`    | When schema inference is enabled, used as custom query to infer it for example if you store multiple entities with different schemas within a container and you want to ensure inference only looks at certain document types|
-| `spark.cosmos.read.inferSchemaSamplingSize`      | `1000`    | Sampling size to use when inferring schema |
+| `spark.cosmos.read.inferSchemaEnabled`     | `true`    | When schema inference is disabled and user is not providing a schema, raw json will be returned. |
+| `spark.cosmos.read.inferSchemaQuery`      | `SELECT * FROM r`    | When schema inference is enabled, used as custom query to infer it. For example, if you store multiple entities with different schemas within a container and you want to ensure inference only looks at certain document types or you want to project only particular columns. |
+| `spark.cosmos.read.inferSchemaSamplingSize`      | `1000`    | Sampling size to use when inferring schema and not using a query. |
```

Comment:
Why would the sampling size only be used when not using a query?

Reply:
Because the query can control the sampling size through `SELECT TOP X...`

Reply:
Tracked by https://github.com/Azure/azure-sdk-for-java/issues/19828